components/raftstore/src/store/txn_ext.rs

```diff
@@ -45,6 +47,19 @@ impl fmt::Debug for TxnExt {
     }
 }
 
+lazy_static! {
+    pub static ref GLOBAL_MEM_SIZE: IntGauge = register_int_gauge!(
+        "tikv_pessimistic_lock_memory_size",
+        "Total memory size of pessimistic locks in bytes."
+    )
+    .unwrap();
+}
+
+const GLOBAL_MEM_SIZE_LIMIT: usize = 100 << 20; // 100 MiB
+
+// 512 KiB, so pessimistic locks in one region can be proposed in a single command.
+const PEER_MEM_SIZE_LIMIT: usize = 512 << 10;
```

Comment:
Are these too small? Is it necessary to make them adjustable?

Reply:
Let's first use a small const, because we don't yet split proposing command before leader transfer.